.github/workflows/terraform-observe_scheduler.yaml

```diff
@@ -27,33 +32,55 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v4
-      - name: Build matrix
-        id: matrix
-        shell: bash
+
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.11'
+
+      - name: download python script
+        id: matrix_v2
```

Comment:
maybe call this `build_matrix` or just `matrix` again.  V2 is a bit confusing since I would assume there was a v1